apm-agent-core/src/main/java/co/elastic/apm/agent/impl/error/ErrorCapture.java

```diff
@@ -238,6 +238,11 @@ public ErrorCapture deactivate() {
          * A hint for UI to be able to show whether a recorded trace for the corresponding transaction is expected
          */
         private boolean isSampled;
+        /**
+         * The related TransactionInfo name
+         */
+        @Nullable
+        private String name;
```

Comment:
What do you think about changing this to a `StringBuilder` so that it can be recycled?

Reply:
So copying the content on `AbstractSpan.getNameForSerialization` to `TransactionInfo.name`?

Reply:
yes, exactly.

Reply:
Done